tests/conftest.py

```diff
@@ -8,3 +8,116 @@
 def test_app():
     client = TestClient(app)
     yield client
+
+
+@pytest.fixture()
+def test_data():
```

Comment:
I don't think these fixtures will be used anywhere but the `test_query` tests for now, no? Is the only reason you're refactoring them here to make the actual test module shorter? I find it a bit less readable to hide them if the only thing we gain from this is a shorter test file

Reply:
Yea kinda. We have quite a few tests and couple fixtures in one module and it seemed cluttered, I thought it's better to move the fixtures to `conftest.py` as I was going to add a few more and make it even more cluttered.
While the pytest docs suggests moving fixtures to `conftest.py` for an entire directory to have access to (which we don't need atm), I think it simplifies the `test_query` module and offers clear test structure.